docs/.vuepress/plugins/generate-nginx-redirects/index.js

```diff
@@ -24,10 +27,12 @@ module.exports = (options, context) => {
     async chainMarkdown() {
       // Workaround for the lack of the hook/callback in the VuePress that is triggered
       // once before the "extendPageData" function.
-      try {
-        await fsp.writeFile(outputFile, fileHead);
-      } catch (ex) {
-        logger.error(`Something bad happens while writing to the file (${outputFile}): ${ex}`);
+      if (DOCS_VERSION === getLatestVersion() || DOCS_VERSION === '**') {
```

Comment:
I think that instead of adding more `if`s maybe it would be a better idea to apply the plugin to the VuePress process only for the latest version. WDYT?
```js
// docs/.config.js
  plugins: [
    // ...
    isLatestVersion() ? [
      nginxRedirectsPlugin, {
        outputFile: path.resolve(__dirname, '../docker/redirects.conf')
      }
    ] : [],
  ]
```

Reply:
Good idea. done